src/Caching/Hybrid/src/HybridCacheOptions.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.Caching.Hybrid;
+
+/// <summary>
+/// Options for configuring the default <see cref="HybridCache"/> implementation
+/// </summary>
+public class HybridCacheOptions
+{
+    /// <summary>
+    /// Default global options to be applied to <see cref="HybridCache"/> operations; if options are
+    /// specified at the individual call level, the non-null values are merged (with the per-call
+    /// options being used in preference to the global options). If no value is specified for a given
+    /// option (globally or per-call), the implementation may choose a reasonable default.
+    /// </summary>
+    public HybridCacheEntryOptions? DefaultEntryOptions { get; set; }
+
+    /// <summary>
+    /// Disallow compression for this <see cref="HybridCache"/> instance
+    /// </summary>
+    public bool DisableCompression { get; set; }
+
+    /// <summary>
+    /// The maximum size of cache items; attempts to store values over this size will be logged.
```

Comment:
Will be logged and then ignored?

Reply:
added "and the value will not be stored in cache"